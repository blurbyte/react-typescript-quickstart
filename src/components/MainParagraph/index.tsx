import styled from '@styles/styledComponents';
import { Color } from '@styles/colors';

const MainParagraph = styled.p`
  font-size: 2.4rem;
  color: ${Color.DarkGray};
  line-height: 1.4;
  margin: 0.8rem 0;
`;

export default MainParagraph;
