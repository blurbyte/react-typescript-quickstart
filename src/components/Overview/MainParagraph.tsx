import styled from '@styles/styledComponents';
import { Colors } from '@styles/colors';

const MainParagraph = styled.p`
  font-size: 2.4rem;
  color: ${Colors.darkGray};
  line-height: 1.4;
  margin: 0.8rem 0;
`;

export default MainParagraph;
