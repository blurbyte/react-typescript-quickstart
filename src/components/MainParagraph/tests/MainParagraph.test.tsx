import * as React from 'react';
import { shallow } from 'enzyme';

import MainParagraph from '../index';

describe('MainParagraph', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<MainParagraph />);
    expect(wrapper).toMatchSnapshot();
  });
});
