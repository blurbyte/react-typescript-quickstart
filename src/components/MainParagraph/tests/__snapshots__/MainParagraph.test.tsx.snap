// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainParagraph renders correctly 1`] = `
.c0 {
  font-size: 2.4rem;
  color: #444444;
  line-height: 1.4;
  margin: 0.8rem 0;
}

<p
  className="c0"
/>
`;
